remo/featuredrep/models.py

```diff
@@ -26,6 +27,14 @@ class Meta:
         permissions = (('can_edit_featured', 'Can edit featured reps'),
                        ('can_delete_featured', 'Can delete featured reps'))
 
+    def save(self, *args, **kwargs):
+        """Override save method for custom functionality"""
+        if not self.updated_on:
```

Comment:
This seems to result in `updated_on` being set when a record is created, and then never set after that, because once `updated_on` is set, this will always return `False`. If the requirement is to update the `updated_on` property every time an update is done, then this will not work.